envision/web/src/components/app.js

```diff
@@ -55,16 +55,19 @@ function App({ client }) {
   const matchedSimulationId = routeMatch ? routeMatch.params.simulation : null;
 
   useEffect(() => {
-    (async () => {
+    const fetchRunningSim = async () => {
       let ids = await client.fetchSimulationIds();
       if (ids.length > 0) {
         if (!matchedSimulationId || !ids.includes(matchedSimulationId)) {
-          history.push(`/${ids[0]}`);
+          history.push(`/${ids[ids.length-1]}`);
```

Comment:
a small improvement: if there are mutiple scenario running. envision displays the first one which is oldest one. choosing the last one displays the latest one.